src/Sdk/Cerbos/Sdk/Cloud/V1/Store/FileOp.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021-2025 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+
+namespace Cerbos.Sdk.Cloud.V1.Store
+{
+    public sealed class FileOp
+    {
+        private Api.Cloud.V1.Store.FileOp.OpOneofCase OneOf = Api.Cloud.V1.Store.FileOp.OpOneofCase.None;
+        private File AddOrUpdate { get; set; }
+        private string Delete { get; set; }
+    
+        private FileOp() {}
+        
+        public static FileOp NewInstance()
+        {
+            return new FileOp();
+        }
+
+        public FileOp OpAddOrUpdate(File addOrUpdate)
```

Comment:
The `Op` prefix is odd, no? Why not just name these methods `AddOrUpdate` and `Delete`.